core/mondoo-github-incident-response.mql.yaml

```diff
@@ -1,6 +1,9 @@
 packs:
   - uid: mondoo-incident-response-github-org
     name: Mondoo GitHub Organization Incident Response
+    tags:
+      mondoo.com/platform: github
```

Comment:
same question as the `cnspec-policies` review I just did...

Do we want to start adding `saas` as a distinct platform?